adodb.inc.php

```diff
@@ -4023,8 +4023,11 @@ class ADORecordSet implements IteratorAggregate {
 	var $_atLastPage = false;	/** Added by Iván Oliva to implement recordset pagination */
 	var $_lastPageNo = -1;
 	var $_maxRecordCount = 0;
+	var $rowsPerPage;
 	var $datetime = false;
 
+	var $oldProvider;
+
```

Comment:
The only reference to this _$oldProvider_ property in the whole code base is in adodb-csvlib.inc.php.

Do you actually need/use this ? If not, I would propose to drop the offending assignment instead of adding a new, useless property in the base class. 

What do you think ?

Reply:
Its been a decade since I've even looked at this code, but you are correct, it seems that $oldProvider is not used anywhere and should probably be removed entirely.

Reply:
Opened #957 to track this